modules/custom-status/custom-status.php

```diff
@@ -282,6 +282,7 @@ public function load_scripts_for_block_editor() {
 		$publish_guard_enabled = ( 'on' === VIP_Workflow::instance()->settings->module->options->publish_guard ) ? true : false;
 
 		wp_localize_script( 'vip-workflow-block-custom-status-script', 'VW_CUSTOM_STATUSES', [
+			'current_user_id'          => get_current_user_id(),
```

Comment:
Should be able to use `wp.data.select("core").getCurrentUser()` instead within the JS code itself so that we let WP/Gutenberg handle it?